04-Enumerable/Enumerable/Task.cs

```diff
@@ -322,13 +362,19 @@ public class Task {
         ///    {(1/1/2010, 10)  , (2/2/2010, 10), (3/3/2010, 10) } => { 30, 0, 0, 0 }
         ///    {(1/1/2010, 10)  , (4/4/2010, 10), (10/10/2010, 10) } => { 10, 10, 0, 10 }
         /// </example>
-        public int[] GetQuarterSales(IEnumerable<Tuple<DateTime, int>> sales) {
+        public int[] GetQuarterSales(IEnumerable<Tuple<DateTime, int>> sales)
+        {
             // TODO : Implement GetQuarterSales
-            throw new NotImplementedException();
+            var firstQuarter = sales.Where(x => x.Item1.Month < 4).Sum(x=>x.Item2);
+            var secondQuarter = sales.Where(x => x.Item1.Month >3&& x.Item1.Month<7).Sum(x => x.Item2);
+            var thirdQuarter = sales.Where(x => x.Item1.Month > 6 && x.Item1.Month < 10).Sum(x => x.Item2);
+            var fourthQuarter = sales.Where(x => x.Item1.Month > 9).Sum(x => x.Item2);
+            int[] result = { firstQuarter, secondQuarter, thirdQuarter, fourthQuarter };
```

Comment:
please try to not combine using var and typed declaration
var here is enough

also it might be just returns in the end without declaration